src/sbvr-api/uri-parser.ts

```diff
@@ -96,12 +96,12 @@ export const memoizedParseOdata = (() => {
 				// Try to cache based on parameter aliases if there might be some
 				const parameterAliases = new URLSearchParams();
 				const queryParams = new URLSearchParams(url.slice(queryParamsIndex));
-				Array.from(queryParams.entries()).forEach(([key, value]) => {
+				for (const [key, value] of Array.from(queryParams.entries())) {
```

Comment:
```suggestion
				for (const [key, value] of queryParams.entries()) {
```

Reply:
I think that's unsafe to do because we use `queryParams.delete(key);` below and that'll affect the iterator if we haven't already pulled it all into an array